roles/edpm_update/molecule/kpatch/converge.yml

```diff
@@ -0,0 +1,56 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
```

Comment:
Can we set gather_facts: false here as well?

Reply:
hmm yep, since the kpatch.yml should ensure we have the needed facts. Good spot, this will ensure we do get the needed fact. Lemme push an update.